### PR TITLE
get-started: add DM encrypt password and update download url

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? 

1. add DM encrypt password 
2. update download url

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/2078/

Note: One of the changed lines (line 232) in this PR overlaps a little with that in https://github.com/pingcap/docs/pull/1653.

### Which version does your change affect? 

dev, 3.0, 3.1, 2.1

### CODE REVIEW

File: dev/how-to/get-started/data-migration.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ For additional information about DM, please consult [Data Migration Overview](/d
 
 We're going to deploy 3 instances of MySQL Server, and 1 instance each of pd-server, tikv-server, and tidb-server. Then we'll start a single DM-master and 3 instances of DM-worker.
 
-First, install MySQL 5.7 and download/extract the TiDB packages we'll use:
+First, install MySQL 5.7 and download/extract the TiDB v3.0 and DM v1.0.2 packages we'll use:
```
Comment (lilin90) [inline dev/how-to/get-started/data-migration.md]:
```suggestion
1. Install MySQL 5.7, download and extract the TiDB v3.0 and DM v1.0.2 packages we'll use:
```

Comment (lilin90) [reply dev/how-to/get-started/data-migration.md]: Suggest making this section into clearer steps.

File: dev/how-to/get-started/data-migration.md [hunk 2]
```diff
@@ -36,13 +36,13 @@ For additional information about DM, please consult [Data Migration Overview](/d
...
 
 ```bash
```
Comment (lilin90) [inline dev/how-to/get-started/data-migration.md]: Indent this code block.

File: v2.1/how-to/get-started/data-migration.md [hunk 3]
```diff
@@ -36,13 +36,13 @@ For additional information about DM, please consult [Data Migration Overview](/v
...
 
-First, install MySQL 5.7 and download/extract the TiDB packages we'll use:
+First, install MySQL 5.7 and download/extract the TiDB v3.0 and DM v1.0.2 packages we'll use:
 
 ```bash
 sudo yum install -y http://repo.mysql.com/yum/mysql-5.7-community/el/7/x86_64/mysql57-community-release-el7-10.noarch.rpm
 sudo yum install -y mysql-community-server
-curl http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz | tar xzf -
-curl http://download.pingcap.org/dm-latest-linux-amd64.tar.gz | tar xzf -
+curl http://download.pingcap.org/tidb-v3.0-linux-amd64.tar.gz | tar xzf -
```
[the remark this answers is not in the document]
Comment (WangXiangUSTC) [reply v2.1/how-to/get-started/data-migration.md]: v2.1 is better

[the remark this answers is not in the document]
Comment (lilin90) [reply v2.1/how-to/get-started/data-migration.md]: @yikeke Please address this comment. Thanks!

**Comment by yikeke** [comment]
Comments addressed. PTAL again. @WangXiangUSTC @lilin90

File: v2.1/how-to/get-started/data-migration.md [hunk 4]
```diff
@@ -36,13 +36,13 @@ For additional information about DM, please consult [Data Migration Overview](/v
 
 We're going to deploy 3 instances of MySQL Server, and 1 instance each of pd-server, tikv-server, and tidb-server. Then we'll start a single DM-master and 3 instances of DM-worker.
 
-First, install MySQL 5.7 and download/extract the TiDB packages we'll use:
+First, install MySQL 5.7 and download/extract the TiDB v3.0 and DM v1.0.2 packages we'll use:
```
Comment (lilin90) [inline v2.1/how-to/get-started/data-migration.md]:
```suggestion
First, install MySQL 5.7 and download/extract the TiDB v2.1 and DM v1.0.2 packages we'll use:
```

File: dev/how-to/get-started/data-migration.md [hunk 5]
```diff
@@ -36,117 +36,126 @@ For additional information about DM, please consult [Data Migration Overview](/d
...
-```
-
-Select the rows back from the MySQL instances to make sure things look right:
-
-```bash
-for i in 1 2 3; do
-    mysql -N -h 127.0.0.1 -P "$((3306+i))" -u root -e 'select * from dmtest1.t1'
-done | sort -n
-```
+1. Create our MySQL database and table in each of the 3 MySQL Server instances:
```
Comment (lilin90) [inline dev/how-to/get-started/data-migration.md]: Please also use ordered steps for other versions. Thanks!

**Pull request merged**